packages/relayer-lib/src/tx_builder/eth_to_cosmos.rs

```diff
@@ -305,6 +311,29 @@ impl<P> TxBuilderService<EthEureka, CosmosSdk> for TxBuilder<P>
 where
     P: Provider + Clone,
 {
+    /// Return relevant metadata for the relayer module.
+    fn metadata(&self) -> HashMap<String, String> {
+        let mut metadata = HashMap::new();
+
+        let update_client_prover_vkey = UpdateClientProgram::get_vkey().bytes32();
+        let membership_prover_vkey = MembershipProgram::get_vkey().bytes32();
+        let uc_and_mem_prover_vkey = UpdateClientAndMembershipProgram::get_vkey().bytes32();
+        let misbehaviour_prover_vkey = MisbehaviourProgram::get_vkey().bytes32();
+
+        metadata.insert(
+            "update_client_prover_vkey".to_string(),
+            update_client_prover_vkey,
+        );
+        metadata.insert("membership_prover_vkey".to_string(), membership_prover_vkey);
+        metadata.insert("uc_and_mem_prover_vkey".to_string(), uc_and_mem_prover_vkey);
```

Comment:
Use `uc_and_membership` or `update_client_and_membership`.

Also `prover_vkey` doesn't make much sense since it is not a prover vkey. Either just use `vkey` or `program_vkey`